index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="stylesheet" href="style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>my-portfolio</title>
+</head>
+<body>
+    <div class="container">
+        <header class="navbar">
+            <a href="#">
+                <img class="my-logo" src="images/My Logo.png" alt="logo">
+            </a>
+            <a href="#">
+                <img class="menu" src="images/union.png" alt="union">
+            </a>
```

Comment:
- [ ] Please use the `nav` tag inside the `header` tag to make your code Semantic. You should wrap your `a` tags with the `nav` tag in order to make your code proper and Semantic. To get more information about Semantic HTML, click this [link](https://www.freecodecamp.org/news/semantic-html5-elements/) 🙏.

Reply:
Well solved

Reply:
Thank you, brother. I am doing my best. I will get there. It is still a lot of details I need to get used to. Thank you again